concordia/views.py

```diff
@@ -682,6 +683,16 @@ def get_context_data(self, **kwargs):
 
         ctx["social_share_flag"] = flag_enabled("SOCIAL_SHARE", request=self.request)
 
+        image_url = asset_media_url(asset)
+        if asset.download_url and "iiif" in asset.download_url:
+            thumbnail_url = asset.download_url.replace(
+                "http://tile.loc.gov", "https://tile.loc.gov"
+            )
+            thumbnail_url = thumbnail_url.replace("/pct:100/", "/!512,512/")
```

Comment:
I'm wondering whether we should use https://github.com/Princeton-CDH/piffle for this — it's more processing to parse the URLs but would give us a higher degree of confidence that it won't break unexpectedly and would be more future-proof for other outside sources.

Reply:
I ticketed that here: https://github.com/LibraryOfCongress/concordia/issues/910